src/migrations/20240206025520-monitoring-data.js

```diff
@@ -0,0 +1,710 @@
+          references: {
+            model: {
+              tableName: 'GrantNumberLinks',
+            },
+            key: 'grantNumber',
+          },
+        },
+        emotionalSupport: {
+          type: Sequelize.DECIMAL(5, 4),
+          allowNull: true,
+        },
+        classroomOrganization: {
+          type: Sequelize.DECIMAL(5, 4),
+          allowNull: true,
+        },
+        instructionalSupport: {
+          type: Sequelize.DECIMAL(5, 4),
+          allowNull: true,
+        },
+        reportDeliveryDate: {
+          type: Sequelize.DATE,
+          allowNull: true,
+        },
+        hash: {
+          type: Sequelize.TEXT,
+          allowNull: true,
+        },
+        sourceCreatedAt: {
+          allowNull: false,
+          type: Sequelize.DATE,
+        },
+        sourceUpdatedAt: {
+          allowNull: false,
+          type: Sequelize.DATE,
+        },
+        sourceDeletedAt: {
+          allowNull: true,
+          type: Sequelize.DATE,
+        },
+        createdAt: {
+          allowNull: false,
+          type: Sequelize.DATE,
+        },
+        updatedAt: {
+          allowNull: false,
+          type: Sequelize.DATE,
+        },
+        deletedAt: {
+          allowNull: true,
+          type: Sequelize.DATE,
+        },
+      }, {
+        transaction,
+      });
+
+      await queryInterface.sequelize.query(/* sql */`
+          CREATE INDEX "MonitoringClassSummaries_reviewId"
+          ON "MonitoringClassSummaries"
+          ("reviewId")
+          WHERE "deletedAt" IS NULL;
+      `, { transaction });
+
+      await queryInterface.sequelize.query(/* sql */`
+          CREATE INDEX "MonitoringClassSummaries_grantNumber"
+          ON "MonitoringClassSummaries"
+          ("grantNumber")
+          WHERE "deletedAt" IS NULL;
+      `, { transaction });
+
+      await queryInterface.sequelize.query(/* sql */`
+          CREATE UNIQUE INDEX "MonitoringClassSummaries_reviewId_grantNumber_deletedAt"
+          ON "MonitoringClassSummaries"
+          ("reviewId", "grantNumber", "deletedAt");
+      `, { transaction });
+
+      await queryInterface.sequelize.query(/* sql */`
+          ALTER TABLE "MonitoringClassSummaries"
+          ADD CONSTRAINT "MonitoringClassSummaries_reviewId_grantNumber_deletedAt_unique"
+          UNIQUE USING INDEX "MonitoringClassSummaries_reviewId_grantNumber_deletedAt";
+      `, { transaction });
+
+      //-----------------------------------------------------------------------------------------
+
+      // All the values read for ENV base on the names
+      const ftpSettings = {
+        host: 'ITAMS_MD_HOST',
+        port: 'ITAMS_MD_PORT',
+        username: 'ITAMS_MD_USERNAME',
+        password: 'ITAMS_MD_PASSWORD',
+      };
+
+      /**
+       * This cron expression breaks down as follows:
+       *  0 - The minute when the job will run (in this case, 0 minutes past the hour)
+       *  21 - The hour when the job will run (in this case, 11 pm)
```

Comment:
2100 is 9:00pm. We should also mention that this is UTC (assuming it is!) thus 21 UTC is 16 EST, meaning 4pm Eastern.